crates/turbopack-node/src/transforms/postcss.rs

```diff
@@ -248,13 +248,18 @@ pub(crate) async fn config_loader_source(
     // Bundling would break the ability to use `require.resolve` in the config file.
     let code = formatdoc! {
         r#"
-            const configPath = `${{process.cwd()}}/{config_path}`;
+            import {{ pathToFileURL }} from 'url';
 
-            const mod = await __turbopack_external_import__(configPath);
+            const configPath = `${{process.cwd()}}/${{{config_path}}}`;
```

Comment:
This would better use `path.join` since the `/` is technically not the correct path separator on windows.

Reply:
@sokra I was trying to keep the change small, and I saw in the node documentation that:

> On Windows, both the forward slash (/) and backward slash (\) are accepted as path segment separators; however, the path methods only add backward slashes (\).

If you'd like, I can submit a follow-up PR.